addons/website/static/src/scss/website.scss

```diff
@@ -1029,6 +1030,11 @@ header {
                         }
```

Comment:
@qsm-odoo 
This code is weird because it only fixes the header outside of edit mode and when the user is logged in .. While it is broken in any case .. I take care of it in this PR, if you agree

Reply:
:+1: